e2e_tests/tests/command/test_notebook.py

```diff
@@ -17,3 +21,31 @@ def test_basic_notebook_start_and_kill() -> None:
             lines.append(line)
 
     raise ValueError(lines)
+
+
+@pytest.mark.e2e_cpu
+def test_notebook_proxy() -> None:
+    session = determined_test_session(ADMIN_CREDENTIALS)
+
+    def get_proxy(session: api.Session, task_id: str) -> None:
+        session.get(f"proxy/{task_id}/")
+
+    typ = NTSC_Kind.notebook
+    created_id = launch_ntsc(session, 1, typ)
+    print(f"created {typ} {created_id}")
+    wait_for_ntsc_state(
+        session,
+        NTSC_Kind(typ),
+        created_id,
+        lambda s: s == bindings.taskv1State.RUNNING,
+        timeout=300,
+    )
+    deets = get_ntsc_details(session, typ, created_id)
+    assert deets.state == bindings.taskv1State.RUNNING, f"{typ} should be running"
+    err = api.task_is_ready(determined_test_session(ADMIN_CREDENTIALS), created_id)
+    assert err is None, f"{typ} should be ready {err}"
+    print(deets)
```

Comment:
nit: can we add some a prefix so readers will know what this is? like `print("ntsc details:", deets)`

Reply:
I didn't mean to leave this print there.